src/mono/wasm/Wasm.Build.Tests/Templates/WasmTemplateTests.cs

```diff
@@ -108,7 +108,7 @@ private async Task BrowserRunTwiceWithAndThenWithoutBuildAsync(Configuration con
         public static IEnumerable<object?[]> BrowserBuildAndRunTestData()
         {
             yield return new object?[] { "", BuildTestBase.DefaultTargetFramework, DefaultRuntimeAssetsRelativePath };
-            yield return new object?[] { "-f net10.0", "net10.0", DefaultRuntimeAssetsRelativePath };
+            yield return new object?[] { $"-f {DefaultTargetFramework}", DefaultTargetFramework, DefaultRuntimeAssetsRelativePath };
 
             if (EnvironmentVariables.WorkloadsTestPreviousVersions)
```

Comment:
I'm a little bit worried that we'll forget updating the earlier versions here.
Should we introduce a `int MinimumTargetFramwork = 8` in BuildTestBase.cs and then here return all TFMs from MinimumTargetFramwork => DefaultTargetFramework?